app/components/Modal/WithdrawModalNew.jsx

```diff
@@ -125,6 +125,7 @@ class WithdrawModalNew extends React.Component {
         let gateFee = 0;
 
         if (selectedGateway) {
+            console.log(selectedGateway);
```

Comment:
Please clean up the code when submitting a PR

Reply:
thx u very match